cmd/node/list_test.go

```diff
@@ -0,0 +1,114 @@
+package node
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/cilium/cilium/api/v1/relay"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cilium/cilium/api/v1/observer"
```

Comment:
nit: we usually rename-import protobuf generated code to <...>pb, (`observerpb` in this case). This prevents confusion with the `observer` package.